app/src/app/[lng]/[inventory]/page.tsx

```diff
@@ -573,125 +585,45 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
-                      Learn more
-                    </Link>{" "}
-                    about GPC Protocol
-                  </Trans>
-                </Text>
-                <Box className="flex w-full justify-between items-center mt-2 gap-6">
-                  <SegmentedProgress
-                    values={[thirdPartyProgress, uploadedProgress]}
-                    colors={["interactive.connected", "interactive.tertiary"]}
-                  />
-                  <Heading
-                    fontWeight="semibold"
-                    fontSize="body.md"
-                    className="whitespace-nowrap"
-                  >
-                    {formatPercent(totalProgress)}%{" "}
-                    <Trans t={t}>completed</Trans>
-                  </Heading>
-                </Box>
-                <Box className="flex gap-4 mt-2">
-                  <Tag>
-                    <TagLeftIcon
-                      as={CircleIcon}
-                      boxSize={6}
-                      color="interactive.connected"
+            <Box className="flex mx-auto max-w-full w-[1090px] css-0">
+              <Tabs align="start" variant="line" isLazy>
+                <TabList>
+                  {[
+                    t("tab-emission-inventory-calculation-title"),
+                    t("tab-emission-inventory-results-title"),
+                  ]?.map((tab, index) => (
+                    <Tab key={index}>
+                      <Text
+                        fontFamily="heading"
+                        fontSize="title.md"
+                        fontWeight="medium"
+                      >
+                        {t(tab)}
+                      </Text>
+                    </Tab>
+                  ))}
+                </TabList>
+                <TabPanels>
+                  <TabPanel>
+                    <InventoryCalculationTab
+                      lng={lng}
+                      inventory={inventory}
+                      inventoryProgress={inventoryProgress}
+                      isUserInfoLoading={isUserInfoLoading}
+                      isInventoryProgressLoading={isInventoryProgressLoading}
                     />
-                    <TagLabel>
-                      {formatPercent(thirdPartyProgress)}%{" "}
-                      <Trans t={t}>connect-third-party-data</Trans>
-                    </TagLabel>
-                  </Tag>
-                  <Tag>
-                    <TagLeftIcon
-                      as={CircleIcon}
-                      boxSize={6}
-                      color="interactive.tertiary"
+                  </TabPanel>
+                  <TabPanel>
+                    <InventoryReportTab
+                      lng={lng}
+                      inventory={inventory}
+                      inventoryProgress={inventoryProgress}
+                      isUserInfoLoading={isUserInfoLoading}
+                      isInventoryProgressLoading={isInventoryProgressLoading}
                     />
```

Comment:
Can we add a <Suspense> around this and make it so it's only loaded once the tab is selected (including all of the required JS and CSS for the charts in the future)?

Reply:
This is taken care of by Chakra, with the isLazy property
https://v2.chakra-ui.com/docs/components/tabs#lazily-mounting-tab-panels